src/pug/guides/code/node/interactive/5.js

```diff
@@ -0,0 +1,42 @@
+const Beam = require('beam-client-node');
+const Tetris = require('beam-interactive-node');
+const rjs = require('robotjs');
+
+const channelId = 1234;
+const username = 'connor';
+const password = 'password';
+
+const beam = new Beam();
+
+beam.use('password', {
+    username,
+    password,
+})
+.attempt()
+.then(() => beam.game.join(channelId))
+.then(res => createRobot(res))
+.then(robot => handleRobotConnection(robot));
+
+function createRobot (res) {
+    return new Tetris.Robot({
+        remote: res.body.address,
+        channel: channelId,
+        key: res.body.key,
+    });
+}
+
+function handleRobotConnection (robot) {
+    robot.handshake(err => {
```

Comment:
Since we are using promises here, it would probably be better to make this return a promise instead and attach a `catch` handler handler in the _bootstrap_ chain.

Could also attach a quick `robot.on('error', ...)` to emphasise that error handling is required there.